### PR TITLE
Inclusion of Nuage-VSP plugin

### PR DESCRIPTION
**Pull request by KrisSterckx**
Documentation for the 4.7.0 Nuage-VSP plugin

### CODE REVIEW

**Comment by KrisSterckx** [comment]
Hi all, @runseb 
what should do I to get attention to this PR ? 

Happy New Year, all !
Kris

File: rtd/source/networking/nuage-plugin.rst [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+.. Note:: There will be only a single domain instance for ``nuagevsp.vpc.domaintemplate.name``.
+
+Networks created in CloudStack will then use domain instances created from the template to which the name points.
+
+
+
+Revision History
+----------------
+
+Fiona Fincannon 21 December 2015
```
Comment (runseb) [inline rtd/source/networking/nuage-plugin.rst]: We don't keep revision history with author info in the actual doc. It's all in git.
Can you remove ?

File: rtd/source/networking/nuage-plugin.rst [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+To add Nuage VSP as Network Service Provider, 
+
+1.  Add the specified network service provider:
+
+::
+
+        http://135.227.147.106:8080/client/api?command=addNetworkSer
+        viceProvider&name=NuageVsp&physicalnetworkid=87528ea5-0189-4
+        a02-92db-3d1539232e21&response=json&sessionkey=CaDCr2P1qpIqm
+        Fsr%2BmMl1T3nLzs%3D&_=1414200788068
```
Comment (runseb) [inline rtd/source/networking/nuage-plugin.rst]: That's weird.
First what is that IP for the mgt server ? it should be more generic.
Second the networkId and session key will be different ....
I suggest giving an example with a cloud monkey CLI.

File: rtd/source/networking/nuage-plugin.rst [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+2.  Add the specified Nuage VSD:
+
+::
+
+    http://135.227.147.106:8080/client/api?command=addNuageVspDe
+    vice&physicalnetworkid=87528ea5-0189-4a02-92db-3d1539232e21&
+    hostname=135.227.210.196&username=cloudstackuser1&password=c
+    loudstackuser1&port=8443&apiversion=v3_2&retrycount=4&retryinter
+    val=60&response=json&sessionkey=CaDCr2P1qpIqmFsr%2BmMl1T3nLz
+    s%3D
```
Comment (runseb) [inline rtd/source/networking/nuage-plugin.rst]: Same here, I'd rather see a cloudmonkey CLI...this is actually unusable for a user .IMHO

**Comment by runseb** [comment]
@KrisSterckx I just replied with a few comments. Once addressed I can merge
Thanks for the docs

**Comment by nlivens** [comment]
@runseb, I've handled your comments

**Comment by runseb** [comment]
LGTM +1 merging

**Comment by KrisSterckx** [comment]
Thanks @nlivens and @runseb !   
Additional question i have : How does this end up now at http://docs.cloudstack.apache.org/en/latest/ ?

thanks!

**Comment by terbolous** [comment]
It is there (http://docs.cloudstack.apache.org/en/latest/networking/nuage-plugin.html) but should be linked to from the index and networking pages.

**Comment by KrisSterckx** [comment]
@terbolous OK thanks - is that something you will take up ? Thanks

**Comment by terbolous** [comment]
already made a PR for it :) #15

**Pull request closed without merging**